kie-api/src/main/java/org/kie/api/executor/ExecutorStoreService.java

```diff
@@ -30,6 +30,13 @@
     
     List<RequestInfo> loadRequests();
 
+    /**
+     * load request that the scheduled timer are older that a certain amount in ms
```

Comment:
It's not in ms but in the unit specified by org.kie.executor.timeunit (if I understood correctly)